src/main/java/org/broadinstitute/dsm/model/gbf/Item.java

```diff
@@ -11,18 +11,20 @@
     private String serialNumber;
     private String expireDate;
     private String shippedQty;
+    private String returnTracking;
     private List<SubItem> subItem;
 
     public Item() {
     }
 
-    public Item(String itemNumber, String lotNumber, String serialNumber, String expireDate, String shippedQty, List<SubItem> subItem) {
+    public Item(String itemNumber, String lotNumber, String serialNumber, String expireDate, String shippedQty, String returnTracking, List<SubItem> subItem) {
         this.itemNumber = itemNumber;
         this.lotNumber = lotNumber;
         this.serialNumber = serialNumber;
         this.expireDate = expireDate;
         this.shippedQty = shippedQty;
-        this.subItem = subItem;
+        this.returnTracking= returnTracking;
+//        this.subItem = subItem;
```

Comment:
sure you want to comment subItems out?

Reply:
There is no sub item in the new confirmation xml. I'll send over their email to yoou

Reply:
then change the constructor as well, and remove it nicely, please